cmd/snap/cmd_snapshot.go

```diff
@@ -410,9 +417,22 @@ func init() {
 				desc: i18n.G("The filename of the export"),
 			},
 		})
-	// This command is hidden because there's no corresponding
-	// "import-snapshot" to consume the produced data.
-	// TODO: implement import-snapshot and remove the hidden attribute.
+	// XXX: this command is hidden because import/export is not complete
+	cmd.hidden = true
+
+	cmd = addCommand("import-snapshot",
+		shortImportSnapshotHelp,
+		longImportSnapshotHelp,
+		func() flags.Commander {
+			return &importSnapshotCmd{}
+		}, nil, []argDesc{
+			{
+				name: "<filename>",
+				// TRANSLATORS: This should not start with a lowercase letter.
+				desc: i18n.G("The filename of the snapshot set export to import"),
```

Comment:
This reads weird, I would say this:
```suggestion
				desc: i18n.G("Name of the snapshot set file to import"),
```